ssn/chapters/Common.html

```diff
@@ -1186,20 +1145,27 @@ <h6><a href="#SOSAExecution">sosa:Execution</a></h6>
             <p class="note">Execution added in the 2024 Update.
                 This term is non-normative pending further implementation experience</p>
             <p>
-                <em property="rdfs:label">Execution of a Procedure</em> -
+                <em property="rdfs:label">Execution</em> -
                 <span property="skos:definition">Act of carrying out a <a href="#SOSAProcedure">Procedure</a>.
-                    This class generalizes the <a href="#SOSAActuation">Actuation</a>, <a
-                        href="#SOSAObservation">Observation</a>, and <a href="#SOSASampling">Sampling</a> classes.
-                    It is not expected to be used directly.</span>
+                </span>
                 <br>
             </p>
             <table>
                 <tbody>
                     <tr>
-                        <th>is Defined By</th>
+                        <th>Scope note</th>
                         <td>
-                            <span rel="rdfs:isDefinedBy"><a
-                                    href="http://www.w3.org/ns/sosa/common/">http://www.w3.org/ns/sosa/common/</a></span>
+                            <span rel="skos:scopeNote">
+                                Execution generalizes the <a href="#SOSAActuation">Actuation</a>, <a
+                                    href="#SOSAObservation">Observation</a>, and
+                                <a href="#SOSASampling">Sampling</a> classes.
+                                It appears in the ontology to support the description of a common pattern for executions
+                                and their properties.
+
+                                Execution may be considered to be abstract, so individuals SHOULD NOT be declared to be
```

Comment:
Consider replacing "may" with "should" or "must". See comments on the TTL.

Reply:
I've made the HTML consistent with RDF/TTL everywhere.